src/components/Tooltip/Tooltip.spec.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import Tooltip from '.';
+
+describe('Tooltip', () => {
+  it('aligns to the center', () => {
+    const component = create(
+      <Tooltip align={Tooltip.Center} content="The tooltip content">
+        Something with tooltip
+      </Tooltip>
+    ).toJSON();
```

Comment:
I think with the emotion serializer, we don't need to call `toJSON()`. It takes care of this for us.